### PR TITLE
Allow to specify the FB API version

### PR DESCRIPTION
**Pull request by jamedranoa**
see https://github.com/lemieux/react-messenger-plugin/issues/11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,10 +32,12 @@ render(
 Expect a delay before the button appears. This is mainly caused by the time required by the Facebook SDK to load and then the time for it to initialize. To reduce the delay, see the advance usage described below.
 
 ### Props
+
 | Name | Type | Required? | Default | Description |
-| --- | --- | ---- | --- | --- | --- |
+| --- | --- | --- | --- | --- |
```
Comment (jamedranoa) [inline README.md:37]: a small change to fix the table format

**Review by lemieux — APPROVED** [review]
All good! thanks for the contribution!

**Comment by lemieux** [comment]
I'll merge this and publish it right away.

**Comment by lemieux** [comment]
Version 2.1.0 is out with your changes!

**Pull request merged**